tools/rapids-bash-functions

```diff
@@ -0,0 +1,3 @@
+function echo_stderr {
+    echo "    [$(basename $0)] $*" >&2
```

Comment:
can you add a comment for why `basename $0` is needed? seems ambiguous to me. this file is ultimately sourced by another file, right? 

maybe add a comment header to this file entirely to state it's intended use